### ISSUE TITLE
Did not detect nodejs 12.x candidate for installation

### ISSUE THREAD

**Issue by cod3r0k** [bigbluebutton/bigbluebutton#12191]
**Describe the bug**
When you want to install BBB 2.3, you encountered with the bellow error and get stuck.

```

Warning: apt-key output should not be parsed (stdout is not a terminal)
Warning: apt-key output should not be parsed (stdout is not a terminal)
Warning: apt-key output should not be parsed (stdout is not a terminal)
bbb-install: Did not detect nodejs 12.x candidate for installation

```

**Comment by ffdixon** [comment]
Are you installing this with `bbb-install.sh`.  Have you encountered this error on previous installations?  If not, is there anything different about the current installation process?

**Comment by hostbbb** [comment]
_=/usr/bin/env
Warning: apt-key output should not be parsed (stdout is not a terminal)
Warning: apt-key output should not be parsed (stdout is not a terminal)
Warning: apt-key output should not be parsed (stdout is not a terminal)
Warning: apt-key output should not be parsed (stdout is not a terminal)
Warning: apt-key output should not be parsed (stdout is not a terminal)
deb [ arch=amd64 ] https://repo.mongodb.org/apt/ubuntu bionic/mongodb-org/4.2 multiverse
Warning: apt-key output should not be parsed (stdout is not a terminal)

I get the same thing...   this server had the dev environment setup. i remember back in december having to upgrade node along the way.  not sure if this is the cause.     I will build a fresh install to see if get error.

it is upgrading fine though at rc-2

**Comment by cod3r0k** [comment]
> Are you installing this with `bbb-install.sh`. Have you encountered this error on previous installations? If not, is there anything different about the current installation process?

Dear @ffdixon , I just run `wget -qO- https://ubuntu.bigbluebutton.org/bbb-install.sh | bash -s -- -v bionic-230 -s bbb.example.com -e notice@example.com  -a -w` in a pure Ubuntu 18.04 server and encountered that issue

**Comment by ffdixon** [comment]
Just tried a similar command on our end and it worked.  If you login to your server, what is the output of `dpkg -l | grep node`.

``# dpkg -l | grep node
ii  nodejs                               12.22.1-1nodesource1                        amd64        Node.js event-based server-side javascript engine
```
What happens if you try to do `sudo apt-get update; sudo apt-get install nodejs`?

**Comment by cod3r0k** [comment]
> dpkg -l | grep node

I doesn't have anything in the output with `dpkg -l | grep node` command.

**Comment by cod3r0k** [comment]
> sudo apt-get update; sudo apt-get install nodejs

Wow, I found my solution. I use DNS dumpers and this leads to this problem. 
I change my DNS setting to `8.8.8.8` and it was interestingly solved ! 

Thanks Fred!

**Comment by karstengo** [comment]
as this is completed anyways. Who encounters still this error it has to be nodejs 12! Not below this version! :-)

I added this as repo and did a apt install nodejs

```
apt -y install curl dirmngr apt-transport-https lsb-release ca-certificates
curl -sL https://deb.nodesource.com/setup_12.x | sudo -E bash -
apt -y install nodejs
```

**Comment by ffdixon** [comment]
Thanks, but this issue was for installing BigBlueButton 2.3 -- which is very old and not recommended.  We recommend installing the latest build of BigBlueButton 2.5 (and soon to be released 2.6).
